### DISCUSSION TITLE
Help: Time Specific CRON Expressions Seems Not Working

### DISCUSSION THREAD

**Discussion by Atanusaha143** [taskiq-python/taskiq#376] [Q&A] [answered]
**Description**
I'm testing the compatibility of [taskiq-faststream](https://github.com/taskiq-python/taskiq-faststream) for a project and ran into an issue while experimenting with typical CRON use cases. It appears that CRON expressions set to trigger at specific times aren't working as expected.

**How to reproduce**

```python
from taskiq_faststream import StreamScheduler
from taskiq.schedule_sources import LabelScheduleSource

from app.background_service.subscriber.test_subscriber import broker

from taskiq_faststream import BrokerWrapper

taskiq_broker = BrokerWrapper(broker)

taskiq_broker.task(
    message={"test-schedule": "schedule test running every day at 11:00 PM"},
    subject="test_schedule_subject",
    schedule=[
        {"cron": "00 23 * * *"},  # At 11:00 PM
    ],
)

scheduler = StreamScheduler(
    broker=taskiq_broker,
    sources=[LabelScheduleSource(taskiq_broker)],
)
```

But the following CRON is working though

```python
taskiq_broker.task(
    message={"task": "schedule test running at every two minute"},
    subject="test_schedule_subject",
    schedule=[
        {"cron": "*/2 * * * *"},  # Runs every two minute
    ],
)
```

**Demo:**

https://github.com/user-attachments/assets/1b3c0fcf-aa3d-4bc5-8e98-74f7f4cee1c0

What might be causing CRON expressions specifying exact times to fail in taskiq-faststream? Are there any known limitations or additional configurations required for time-specific scheduling to work as expected?

**Comment by s3rius** [comment, marked as answer]
It can be that your time offset is not the same with UTC. There's a [paragraph](https://taskiq-python.github.io/guide/scheduling-tasks.html#working-with-timezones) about timezones on docs.  You can cron_offset to verify that it works.
